### ISSUE TITLE
Open Graph Missing Support for og:image:alt

### ISSUE THREAD

**Issue by cogdog** [Automattic/jetpack#10593]
#### Is your feature request related to a problem? Please describe.
The Open Graph tags published by JetPack lack data for `og:image:alt` which [according to the og protocol:](http://ogp.me/#structured)

> **og:image:alt** A description of what is in the image (not a caption). If the page specifies an og:image it should specify og:image:alt.

The JetPack code writing og tags is specifying an og:image but not og:image:alt and it sure seems like it should.

This means images shared via social media that use OpenGraph are not providing alternative text for images. 

This came up in [a blog post I made about problems with the implementation of image alternate descriptions in Twitter](http://cogdogblog.com/2018/11/regular-habit-alting/) - I noticed my own blog post tweeted via the Publicize module and using the JetPack Open Graph tags, were missing this element. The featured post in my image has alt text, but no representating in og tags, so when tweeted by Publicize, there is no alt="...." for the image tag.

#### Describe the solution you'd like
Use the `alt text` value of the image described by the media referenced by `og:image` and add og tags for `og:image:alt`

#### Describe alternatives you've considered
I could probably figure out my own hook and fix my site, but this one seems pretty doable by the JetPack pros so everyone else can have it too.

**Comment by cogdog** [comment]
Bueller? Bueller? Is this where issues go to die?

**Comment by jeherve** [comment]
This feature is not on our roadmap right now, but I labelled it "Good for Community"; if you'd like to contribute a PR with that improvement, we'd be happy to review it!
